keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2564,7 +2564,7 @@
           "additionalProperties": {
             "type": "string"
           },
-          "description": "Application specific metadata in the form of key-value pairs."
+          "description": "Application specific metadata in the form of key-value pairs. The element keyword and braces, ‘{‘ ‘}’, are required even if there are no attributes."
```

Comment:
The sentence you added seems to be REST API specific. Doesn't apply to the generated code.